Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -115,5 +120,39 @@ public void Save()
         {
             _storage.Save();
         }
+
+        public List<Result> LoadContextMenus(Result selectedResult)
+        {
+            return new List<Result>() {
+                new Result
+                {
+                    Title = context.API.GetTranslation("flowlauncher_plugin_browserbookmark_copyurl_title"),
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_browserbookmark_copyurl_subtitle"),
+                    Action = _ =>
+                    {
+                        try
+                        {
+                            Clipboard.SetText(((BookmarkAttributes)selectedResult.ContextData).Url);
```

Comment:
please use SetDataObject due to some copy issue mentioned before.

Reply:
Yes, you right, forgot about that.

Reply:
Thank you!